src/Services/Order/OrderImportService.php

```diff
@@ -94,7 +112,15 @@ public function run($from, $to)
 							->addReference('etsyReceiptId', $receiptData['receipt_id'])
 							->report('Etsy::order.startOrderImport', $receiptData);
 
-						$this->orderCreateService->create($receiptData);
+						$lang = '';
```

Comment:
is there maybe a useful way to implement a fallback value or so $lang cannot be an empty string or do you now what will happen if it is an empty string?

Reply:
I would say lets use english.